test/simulator/moves/clangoroussoulblaze.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+const assert = require('./../../assert');
+const common = require('./../../common');
+
+let battle;
+
+describe('Z-Moves', function () {
+	afterEach(function () {
+		battle.destroy();
+	});
+
+	it(`should deal reduced damage to only protected targets`, function () {
+		battle = common.createBattle({gameType: 'doubles'});
+		battle.join('p1', 'Guest 1', 1, [
+			{species: 'Kommo-o', ability: 'overcoat', item: 'kommoniumz', moves: ['clangingscales']},
+			{species: 'Pachirisu', ability: 'voltabsorb', moves: ['protect']},
+		]);
+		battle.join('p2', 'Guest 2', 1, [
+			{species: 'Tapu Lele', ability: 'psychicsurge', moves: ['protect']},
+			{species: 'Zapdos', ability: 'pressure', moves: ['tailwind']},
+		]);
+		battle.makeChoices("move clangingscales zmove move protect", "move protect move tailwind");
```

Comment:
COMMAS

```
battle.makeChoices("move clangingscales zmove, move protect", "move protect, move tailwind");
```

Literally the only reason this works is because it's ignoring everything and using the default choices.

Reply:
Heh, I actually noticed this before merging, but since the test passed I figured it worked anyway. Fixed in https://github.com/Zarel/Pokemon-Showdown/commit/4d3440fedb8ee7e956dfb350bf1cea01e63d5c5f!